plugins/source/aws/resources/services/s3/buckets_fetch.go

```diff
@@ -136,11 +137,12 @@ func resolveS3BucketsAttributes(ctx context.Context, meta schema.ClientMeta, res
 func fetchS3BucketGrants(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
 	r := parent.Item.(*models.WrappedBucket)
 	svc := meta.(*client.Client).Services().S3
-	if parent.Get("region").(string) == "" {
```

Comment:
Similar to before, maybe `r.Region` will work for these? Or we do some processing on it?

Reply:
I think I just improved the situation by knowing for sure what will be the type so if any changed I suggest to do follow-up PRs. before that we assumed it was string and it could fail in runtime. now we ensure it is always `*cqtypes.Text`